mapper/orm-coordination-outbox-polling/src/main/java/org/hibernate/search/mapper/orm/coordination/outboxpolling/cfg/HibernateOrmMapperOutboxPollingSettings.java

```diff
@@ -445,15 +445,30 @@ private HibernateOrmMapperOutboxPollingSettings() {
 			PREFIX + Radicals.COORDINATION_ENTITY_MAPPING_AGENT_TABLE;
 
 	/**
-	 * The name of the identifier generator used for the agent table.
+	 * The name of UUID generator strategy to be used by {@link UUIDGenerator}.
+	 * <p>
+	 * Only available when {@value HibernateOrmMapperSettings#COORDINATION_STRATEGY} is
+	 * {@value HibernateOrmMapperOutboxPollingSettings#COORDINATION_STRATEGY_NAME}.
+	 * <p>
+	 * The default for this value is {@link  Defaults#COORDINATION_ENTITY_MAPPING_AGENT_UUID_GEN_STRATEGY}.
+	 *
+	 * @see UUIDGenerator
+	 * @see CustomVersionOneStrategy
+	 * @see StandardRandomStrategy
+	 */
+	public static final String COORDINATION_ENTITY_MAPPING_AGENT_UUID_GEN_STRATEGY =
+			PREFIX + Radicals.COORDINATION_ENTITY_MAPPING_AGENT_UUID_GEN_STRATEGY;
+
+	/**
+	 * The database type used for representing an UUID in the agent table.
```

Comment:
```suggestion
	 * The JDBC type used for representing an UUID in the agent table.
```

Reply:
now I'm wondering if we maybe should say that it's actually a `name of an ORM registered basic type` or something along that? And we probably want to apply it to the outbox event doc as well as to the reference docs to be consistent...

Reply:
If you find the correct name, sure. But it needs to be the official way of referring to those types in Hibernate ORM; I'm not sure "basic type" is the sanctioned wording?

Reply:
added this as an additional commit so it's easier to see where it got changed